docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-assign-task-to-role-proc.adoc

```diff
@@ -1,2 +1,23 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-assign-task-to-role-proc-{context}']
+= Assigning a task to a role 
+
+Case management processes need to be a flexible as possible to accommodate changes that can happen dynamically during run time. This includes changing user assignments for new case instances or for active cases. For this reason, it is important that roles are not hardcoded to a single set of users or groups in the case definition. Instead, role assignments can be defined on the task nodes in the case definition, with users or groups assigned to the roles on case creation.  
```

Comment:
"be as flexible as". Also, I try to avoid empty "it is", "there is/are", etc. for clarity and accessibility. So maybe "For this reason, make sure that you do not hard-code... Instead, define role assignments on ..." Or just "make sure roles are not hard-coded" or whatnot if passive sounds better. I just go for active as much as I can. Anyway, not critical and I don't care that much. Just a thought :)

Reply:
ack. Fixed.